src/main/java/codeu/controller/RoutingEngine.java

```diff
@@ -0,0 +1,19 @@
+package codeu.controller;
+
+import codeu.model.data.BotRequest;
+import codeu.model.data.RocketBotHandler;
+	
+public class RoutingEngine {
+	
+	//primary routing function that passes the request to the appropriate Handler
+	//needs to be updated for each bot
+	public String routeCommand(BotRequest request) {
+		
+		switch (request.getCommand()) {
+		case "rocket":
```

Comment:
We should probably require our bots to have static string members so you can do:
```
switch (request.getCommand()) {
case RocketBotHandler.COMMAND_NAME:
}
```